### PR TITLE
bug 1238587 - Quotes, comments, periods, and whitespace

### PR DESCRIPTION
**Pull request by jwhitlock**
Bump requirements to make requires.io momentarily happy, and to update for a Django security fix.  Then enable some additional flake8 checks for ``make lint``:

* ``D100`` - Add docstrings for public modules
* ``D104`` - Add docstrings to ``__init__.py`` files
* ``D204`` - Add 1 blank line after class docstrings
* ``D211`` - Remove blank lines before class docstrings
* ``D400`` - Convert first docstring lines to proper statements
* ``Q000`` - (new with ``flake8-quotes``) - Changed double quotes to single quotes as needed

Also, installed and enabled ``nose-ignore-docstring``, so that docstrings on test methods won't be used instead of the class and method names, and we can start adding docstrings to test methods to work on  the 1222 instances of ``D102``, "Missing docstring in public method".

### CODE REVIEW

**Comment by jwhitlock** [comment]
Force-pushed to update commit message on last commit, which tells TravisCI to notify IRC when the build breaks.

File: wpcsite/settings.py [hunk 1]
```diff
@@ -32,7 +32,7 @@
     BROKER_URL is undefined, or 0 if defined
 CELERY_RESULT_BACKEND - Backend URL string for Celery
 DATABASE_URL - See https://github.com/kennethreitz/dj-database-url
-DEFAULT_FROM_EMAIL - "From" email for emails to users
+DEFAULT_FROM_EMAIL - 'From' email for emails to users
```
Comment (willkg) [inline wpcsite/settings.py]: I think in prose, I'd stick with " rather than '. I don't think it matters a whole lot, though.

**Comment by willkg** [comment]
I skimmed the changes and they look fine to me. r+!

**Comment by jwhitlock** [comment]
Force-push to use double-quotes in the prose section of ``wpcsite/settings.py``. Will merge when Travis is happy.

**Pull request merged**